lib/puppet/parser/functions/lastpass_functions.rb

```diff
@@ -110,6 +110,8 @@ def get_item_by_uniquename(uniquename)
 
   raise Puppet::ParseError, "error: lpass show [uniquename: #{uniquename}]: #{error}" \
     unless status.success?
+  raise Puppet::ParseError, "error: lpass show [uniquename: #{uniquename}]: Multiple matches found" \
```

Comment:
I assume from this that the command status is still successful which is why this separate condition is required. Could you add a comment stating that so it's clear in the code?

Reply:
Yea the exit status is 0:
```shell
[root@uitlpupp10  PRD  ~]# lpass show --sync=auto "Shared-production-Puppet/core\root/drylmdbp04.mcs.miamioh.edu_password" "--format=%fn<==>%fv"
Multiple matches found.
Shared-production-Puppet/core\root/drylmdbp04.mcs.miamioh.edu_password [id: 2067238960219493786]
Shared-production-Puppet/core\root/drylmdbp04.mcs.miamioh.edu_password [id: 7809189455575403382]
[root@uitlpupp10  PRD  ~]# echo $?
0
```

Reply:
so parse_item chokes with a very unhelpful message